src/main/kotlin/com/dansplugins/factionsystem/command/faction/info/MfFactionInfoCommand.kt

```diff
@@ -0,0 +1,47 @@
+package com.dansplugins.factionsystem.command.faction.info
+
+import com.dansplugins.factionsystem.MedievalFactions
+import com.dansplugins.factionsystem.player.MfPlayer
+import dev.forkhandles.result4k.onFailure
+import org.bukkit.ChatColor.RED
+import org.bukkit.ChatColor.AQUA
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import java.util.logging.Level.SEVERE
+
+class MfFactionInfoCommand(private val plugin: MedievalFactions) : CommandExecutor {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+        if (!sender.hasPermission("mf.info")) {
+            sender.sendMessage("$RED${plugin.language["CommandFactionInfoNoPermission"]}")
+            return true
+        }
+        
+        if (sender !is Player) {
+            sender.sendMessage("$RED${plugin.language["CommandFactionInfoNotAPlayer"]}")
+            return true
+        }
+        plugin.server.scheduler.runTaskAsynchronously(plugin, Runnable {
+            // get player's faction
+            val playerService = plugin.services.playerService
+            val mfPlayer = playerService.getPlayer(sender)
+                ?: playerService.save(MfPlayer.fromBukkit(sender)).onFailure {
+                    sender.sendMessage("$RED${plugin.language["CommandFactionInfoFailedToSavePlayer"]}")
+                    plugin.logger.log(SEVERE, "Failed to save player: ${it.reason.message}", it.reason.cause)
+                    return@Runnable
+                }
+            val factionService = plugin.services.factionService
+            val faction = factionService.getFaction(mfPlayer.id)
+            if (faction == null) {
+                sender.sendMessage("$RED${plugin.language["CommandFactionInfoMustBeInAFaction"]}")
+                return@Runnable
+            }
+            // send player faction info
+            sender.sendMessage("$AQUA${plugin.language["CommandFactionInfoTitle", faction.name]}")
```

Comment:
I think we could make these fields modifiable here - perhaps adding an edit button chat component that, when clicked, opens a conversation that sets the name.
We could make the rename and description commands use a conversation when no arguments are specified and then use that.

Reply:
That sounds good! I haven't used chat components in the past, but we can make an issue to implement this in the future.

I really like using a conversation when no arguments are specified. That's a fantastic idea.

Reply:
I've opened #1500 for this.